src/main/java/core/basesyntax/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandlerService;
+import core.basesyntax.service.OperationStrategyService;
+import java.util.HashMap;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategyService {
+    private final Map<FruitTransaction.Operation,
+            OperationHandlerService> operationHandleMap = new HashMap<>();
+
+    public OperationStrategyImpl() {

```

Comment:
Pass map in constructor, it would make your strategy more flexible if you would want to have other strategy with different list of handlers.